bin/circleci/run-integration-tests.sh

```diff
@@ -8,21 +8,18 @@
 #  * disabling these now lets us land greenkeeper and update a lot of out of
 #    date libraries (~25)
 #  * r=_6a68 and r=clouserw.  Conveniently discussed over the holiday break ;)
-exit 0
 
 npm start &
 STATIC_SERVER_PID=$!
 
 # Wait until the server is available...
-until $(curl --output /dev/null --silent --head --fail http://testpilot.dev:8000); do
+until $(curl --output /dev/null --silent --head --fail -k https://example.com:6678); do
```

Comment:
What's expected to start on port 6678? The server started with gulp via `npm start` is still on 8000 - I don't see anything in this PR that changes it to 6678. And, I see this in my test output:
```
[18:10:00] Server started https://0.0.0.0:8000
```